tools/build.sh

```diff
@@ -34,12 +34,19 @@ fi
 
 echo "Updating scripts"
```

Comment:
Great, you found the right spots to do this :-) At some point we may want to extract this into a common place so both `update.sh` and `build.sh` can call the same bit of code.